spec/support/features/webauthn_helper.rb

```diff
@@ -127,6 +127,15 @@ def set_hidden_field(id, value)
     end
   end
 
+  def simulate_platform_authenticator_available
+    page.evaluate_script(<<~JS)
+      window.PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable = () => Promise.resolve(true);
+    JS
+    page.evaluate_script(<<~JS)
+      document.querySelectorAll('lg-webauthn-input').forEach((input) => input.connectedCallback());
+    JS
```

Comment:
Oof, I'd forgot about needing this. I think it makes sense, though I'd be a little concerned about future maintainability if the connected callback has some side effects that might be doubled-up by calling it explicitly like this.

In the past I'd wondered if we could use the [Chrome DevTools protocol support for WebAuthn](https://chromedevtools.github.io/devtools-protocol/tot/WebAuthn/) to add a "virtual" authenticator, but it was experimental and I had a hard time getting it working. Might be something to reconsider in the future.